webview-ui/src/RetinaCapture/RetinaCapture.tsx

```diff
@@ -0,0 +1,53 @@
+import { InitialState } from "../../../src/webview-contract/webviewDefinitions/retinaCapture";
+import { VSCodeButton, VSCodeDivider } from "@vscode/webview-ui-toolkit/react";
+import { useStateManagement } from "../utilities/state";
+import { stateUpdater, vscode } from "./state";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import styles from "./RetinaCapture.module.css";
+
+export function RetinaCapture(initialState: InitialState) {
+    const { state } = useStateManagement(stateUpdater, initialState, vscode); //eventHandlers
+
+    function handleNodeExplorerPod(nodeName: string) {
+        vscode.postRunRetinaCapture(nodeName);
+        document.getElementById(nodeName)?.showPopover();
```

Comment:
The 'React' way of doing this is to use [refs](https://react.dev/learn/manipulating-the-dom-with-refs). This would avoid the need for `id` attributes on the elements.

But actually, I can't see where the popover is being created here. Does this do anything?